### PR TITLE
release: v0.6.0

### PR DESCRIPTION
**Pull request by mihaibivol**

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -11,25 +11,33 @@ Dictdiffer is a helper module that helps you to diff and patch
...
+
+-- Updates package structure, sorts imports and runs doctests.
+
+Bug fixes
+~~~~~~~~~
 
-- Adds support for comparing numerical values and NaN. (#54) (#55)
+-- Fixes order in which handled conflicts are unified so that the
+- Merger's unified_patches can be always applied.
+-- Removes unused `Conflict.handled` field.
```
Comment (tiborsimko) [inline RELEASE-NOTES.rst]: The `Removes unused...` part should actually not be advertised, I guess `kwalitee prepare release` does not do the parsing job here correctly... We usually fix it by introducing empty `AMENDS` commits, see for example https://github.com/inveniosoftware/flask-cli/pull/13#issuecomment-219976929

Comment (mihaibivol) [inline RELEASE-NOTES.rst]: I'll just keep the first bullet then.

**Comment by mihaibivol** [comment]
@tiborsimko I updated the PR with cleaner RELEASE-NOTES

**Comment by jirikuncar** [comment]
Please fix the typo in commit message.

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -11,25 +11,32 @@ Dictdiffer is a helper module that helps you to diff and patch
 dictionaries.
 
 New features
-------------
+~~~~~~~~~~~~
```
Comment (jirikuncar) [inline RELEASE-NOTES.rst]: keep -

**Comment by mihaibivol** [comment]
Done

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -13,23 +13,30 @@ dictionaries.
 New features
 ------------
 
-- Adds tolerance parameter used when user wants to treat closed values
-  as equals
+-- Adds support for comparing NumPy arrays.  (#68)
```
Comment (jirikuncar) [inline RELEASE-NOTES.rst]: This is probably a typo. Can you check the output of `kwalitee prepare release v0.5.0..`?

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -13,23 +13,30 @@ dictionaries.
...
+-- Adds support for comparing mutable mappings, sequences and sets from
+-  `collections.abs` module.  (#67)
 
 Improved features
 -----------------
 
-- Adds support for comparing numerical values and NaN. (#54) (#55)
```
Comment (mihaibivol) [inline RELEASE-NOTES.rst]: @jirikuncar The `kwalitee` tool gave only one dash, but I added two to be consistent with the previous version.

Comment (jirikuncar) [inline RELEASE-NOTES.rst]: It was probably part of the diff output ...

Comment (mihaibivol) [inline RELEASE-NOTES.rst]: Oh, this was silly of me :disappointed:

**Comment by tiborsimko** [comment]
> @tiborsimko I updated the PR with cleaner RELEASE-NOTES

We do this via `AMENDS` technique, so that the notes can be really generated automatically, not manually. This helps for the future fully-automated project web site dashboard. Just in case you do it next time...

**Comment by tiborsimko** [comment]
... here is an explicit example how to amend a few messages before releasing Invenio 2.0.4:

```
commit 6f6b0d01b734e438a9b758db363a884697c79b80
Author: Tibor Simko <tibor.simko@cern.ch>
Date:   Mon Jun 1 14:51:14 2015 +0200

    Invenio v2.0.4
    
    Signed-off-by: Tibor Simko <tibor.simko@cern.ch>
    Reviewed-by: Jiri Kuncar <jiri.kuncar@cern.ch>

commit c9dbe0613e31595a643dd362ab23270601da6ac1
Author: Tibor Simko <tibor.simko@cern.ch>
Date:   Mon Jun 1 13:18:29 2015 +0200

    knowledge: fix HTML character encoding
    
    * FIX Fixes HTML character encoding in admin templates. (closes #3118)
    
    * AMENDS acf26cdeea82c4af39b157fe96a9be5c4f82c72f
    
    Signed-off-by: Tibor Simko <tibor.simko@cern.ch>

commit e09e314123d6aaa43e88106eabc115401939a05e
Author: Tibor Simko <tibor.simko@cern.ch>
Date:   Mon Jun 1 13:16:08 2015 +0200

    installation: Apache 2.4 configuration syntax fix
    
    * FIX Fixes syntax error in generated Apache virtual host configuration.
    
    * AMENDS cc4b608cb5a8aeaf6b4df9a0652b006925d7aceb
    
    Signed-off-by: Tibor Simko <tibor.simko@cern.ch>
```

**Pull request merged**